.github/workflows/build-images-nightly.yaml

```diff
@@ -25,3 +25,4 @@ jobs:
       limitadorOperatorVersion: ${{ vars.LIMITADOR_OPERATOR_SHA }}
       dnsOperatorVersion: ${{ vars.DNS_OPERATOR_SHA }}
       wasmShimVersion: ${{ vars.WASM_SHIM_SHA }}
+      quayImageExpiry: 2w
```

Comment:
Abitary value of when to expire nighty images. If anyone has a better suited value, I can update it to this value

Reply:
Maybe @jsmolar (QE) can recommend a value that would work well

Reply:
1 or 2 weeks for nightlies is reasonable. I don't see any reason why we would need to keep them longer.